src/EnergyPlus/AirflowNetwork/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-cmake_minimum_required(VERSION 2.8.12)
-
-project(AirflowNetwork)
-
```

Comment:
The alternative is to put `cmake_policy(SET CMP0054 NEW)` but since this seem to depend on src/EnergyPlus stuff I suppose you won't be building the AirflowNetwork/ subdir by itself (?)

Reply:
Since I do build AirflowNetwork separately, no, this is not a change I can agree with until I can look into more. I'm out of the office right now and only have my phone, so getting up to speed will probably have to wait until next week.

Reply:
This one is a hard no 👎

Reply:
OK, @jmarrec, given that this does get built standalone, could you propose an alternative solution to the issue this change is trying to solve?

Reply:
I did some testing here - it looks like the change to the `if` statement below is enough to make the warning go away. I can revert the two lines above unless there's a different approach you all would rather pursue here?

Reply:
My testing showed the same. My vote is revert.

Reply:
I explicitly pointed to it on purpose

Just put back project, set cmp0054, and keep the changes to the if statements.
I suggest bumping the cmake minimum version here to 3.1, otherwise you have to protect the cmp0054 with an if statement, there is an example in https://github.com/NREL/EnergyPlus/blob/263e15293ef34d7ad2e6bccf57220964f8323933/CMakeLists.txt#L14-L16

Reply:
Thanks @jmarrec, I'm happy to implement these changes. @jasondegraw do you have a need to keep the original CMake version or can we bump it to 3.1 as suggested?

Reply:
@mitchute Sure, that's fine. The CMake version can probably keep up with whatever the main build is at, looks like 3.17? Let's not do this again next week.